CHANGELOG.md

```diff
@@ -25,7 +30,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Migrate model display names from locale/en.yml to plugin [(#42)](https://github.com/ManageIQ/manageiq-providers-google/pull/42)
 
 ### Fixed
-- Add back missing IP address range in Virtual Private Cloud name. [(#46)](https://github.com/ManageIQ/manageiq-providers-google/pull/46)
```

Comment:
hi @JPrause - Im curious why this was moved ?

Reply:
@JPrause - bump

Reply:
Satoe keeps a spreadsheet of all backports per release.
In this instance, when gaprindashvili was released, that particular fix went from "unreleased" to "released" in gap-2.
Not all fixes are backported to the upstream release. That's where her spreadsheet comes in.